src/components/BulkEnrollmentPage/stepper/DismissibleCourseWarning.jsx

```diff
@@ -8,16 +8,14 @@ import { WARNING_ALERT_TITLE_TEXT, WARNING_ALERT_BODY_TEXT } from './constants';
  * Displays simple dismissible banner to remind admins.
  */
 const DismissibleCourseWarning = () => {
-  const [closed, setClosed] = useState(false);
-
-  if (closed) { return null; }
-
+  const [isOpen, setIsOpen] = useState(true);
   return (
     <Alert
```

Comment:
I realized some of our dismissible `Alert` components are not actually dismissible, such as this one. The `dismissible`  prop only controls whether the "Dismiss" button is shown. The actual dismiss behavior is controlled by the `show` and `onClose` props and some state.

I fixed it here, and added an item to the tech debt tracker.